src/Core/src/Platform/Android/ListViewExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Android.Widget;
+using ALayoutDirection = Android.Views.LayoutDirection;
+using ATextDirection = Android.Views.TextDirection;
+
+namespace Microsoft.Maui
+{
+	public static class ListViewExtensions
+	{
+		public static void UpdateFlowDirection(this ListView platformView, IView view)
+		{
+			switch (view.GetEffectiveFlowDirection())
```

Comment:
Why do we need to use GetEffectiveFlowDirection() here? The flow direction of the parent should be available for the ListView.

Reply:
FlowDirection's value here is MatchParent, and it uses the Inherit value of LayoutDirection which leaves the ListView UI unchanged.